src/main/java/jenkinsci/plugins/influxdb/generators/SonarQubePointGenerator.java

```diff
@@ -147,6 +148,7 @@ private void setSonarDetails(String sonarBuildLink) {
                     .addField(SONARQUBE_LINES_TO_COVER, getSonarMetric(sonarMetricsUrl, SONARQUBE_LINES_TO_COVER))
                     .addField(SONARQUBE_DUPLICATED_LINES_DENSITY, getSonarMetric(sonarMetricsUrl, SONARQUBE_DUPLICATED_LINES_DENSITY))
                     .addField(SONARQUBE_COMPLEXITY, getSonarMetric(sonarMetricsUrl, SONARQUBE_COMPLEXITY))
+                    .addField(SONARQUBE_ALERT_STATUS, getSonarMetric(sonarMetricsUrl, SONARQUBE_ALERT_STATUS))
```

Comment:
This will never return anything. `getSonarMetric` returns a `Float` and the `alert_status` is a `String`.

Reply:
Nice catch. My fault.